src/aiida/orm/computers.py

```diff
@@ -561,7 +561,9 @@ def is_configured(self) -> bool:
 
         :return: Boolean, ``True`` if the computer is configured for the current default user, ``False`` otherwise.
         """
-        return self.is_user_configured(users.User.get_collection(self.backend).get_default())
+        user = users.User.get_collection(self.backend).get_default()
+        assert user is not None
```

Comment:
This is a common problem around the codebase -- that we don't check that the default user is set. We should probably create a custom exception for this case and reuse it in all the relevant places. Suggestions welcome.